source/guides/dashboard/runs.md

```diff
@@ -121,3 +121,14 @@ Runs that have been canceled or are in an errored state can be archived from the
   `https://dashboard.cypress.io/projects/{project ID}/runs/{run number}`
 - Click **{% fa fa-history %} Restore from archive**
   {% imgTag /img/dashboard/restore-from-archive.png "restore-from-archive" %}
+
+## Test Case History
```

Comment:
```suggestion
# Test Case History
```

Pushing this up so it's not nested under Archive run

Reply:
Good idea!